app.js

```diff
@@ -1,8 +1,17 @@
 // import functions and grab DOM elements
+import { fetchFamily } from './fetch-utils.js';
+import { renderDetails } from './render-utils.js';
 
-// let state
+const familyList = document.getElementById('family-list');
+// set event listeners
+// eslint-disable-next-line space-before-function-paren
+window.addEventListener('load', async () => {
+    const members = await fetchFamily();
+    // eslint-disable-next-line no-console
+    console.log(members);
 
-// set event listeners 
-  // get user input
-  // use user input to update state 
-  // update DOM to reflect the new state
+    for (let member of members) {
```

Comment:
The rubric calls for this to be pulled into a separate function - renderThing. This for loop would go inside that function and then be called here